tests/fixtures_server.py

```diff
@@ -123,51 +123,69 @@ def get_patched_settings(temp_path: Path):
     return settings
 
 
+modules_to_patch = [
```

Comment:
This list is a nice improvement on the hard-coding of all module names.
Two comments:
1. Please add an explanatory comment, or turn it into a function with an informative docstring. In the current form, it's hard to guess what's happening.
2. The explanation should also say why we exclude modules that contain "migrations".
3. Is there a reason for not checking that "get_settings" is in the file? It would cost a bit more (you would need to read the text file) but it would also reduce the number of patched functions. And it would maybe also make the goal of this block more transparent. Let's maybe first check which fraction of modules do have a match.

Reply:
> Is there a reason for not checking that "get_settings" is in the file? It would cost a bit more (you would need to read the text file) but it would also reduce the number of patched functions. And it would maybe also make the goal of this block more transparent. Let's maybe first check which fraction of modules do have a match.

From Yuri: yes, it's much faster to try the patch and handle the error. And this is also done automatically by pytest, via `raising=False` (see docs here: https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.MonkeyPatch.setattr).